rust/blockstore/src/arrow/blockfile.rs

```diff
@@ -179,13 +231,11 @@ impl ArrowBlockfileWriter {
         // Add the key, value pair to delta.
         // Then check if its over size and split as needed
         delta.add(prefix, key, value);
-        if delta.get_size::<K, V>() > self.block_manager.max_block_size_bytes() {
-            let new_blocks = delta.split::<K, V>(self.block_manager.max_block_size_bytes());
```

Comment:
I don't understand how this was working before--doesn't the old (pre-split) delta need to be removed?

Reply:
The delta is reused. So if you split (A) you get (A, B). Not (B, C)

Reply:
oh right, that makes sense
thanks